### PR TITLE
Pr/link bd to pt

### PR DESCRIPTION
**Pull request by djgribben**
Interfaces bath_dynamics class with compute_correlations function in contractions.py. TODO: Make a nice example.

* [x] The contribution has been discussed and agreed on in the [Issue section](https://github.com/tempoCollaboration/TimeEvolvingMPO/issues).
* [x] Code contributions do its best to follow [the zen of python](https://www.python.org/dev/peps/pep-0020/).
* [x] The automated test are all positive:
  - [x] `tox -e py36` (to run `pytest`) the code tests.
  - [x] `tox -e style` (to run `pylint`) the [code style](https://www.python.org/dev/peps/pep-0008/) tests.
  - [x] `tox -e docs` (to run `sphinx`) generate the documentation.
* [ ] Added test for changed/added code.
* [ ] API code contributions include input checks.
* [ ] API code contributions include helpful error messages.
* [ ] The documentation has been updated:
  - [x] docstring for all functions/methods/classes/modules.
  - [x] consistant style of all docstrings.
  - [ ] for new modules: `/docs/pages/modules.rst` has been updated.
  - [ ] for api contributions: `/docs/pages/api.rst` has been updated.
  - [ ] for api contributions: tutorials and examples have been updated.

### CODE REVIEW

File: oqupy/bath_dynamics.py [hunk 1]
```diff
@@ -91,47 +119,70 @@ def bath_occupation(self,
...
-        correlation_set = [(x,y) for y in range(current_corr_dim,corr_mat_dim)\
-                           for x in range(y+1)]
-        if len(correlation_set)>0:
-            dim_diff = corr_mat_dim-self._system_correlations.shape[0]
-            coup_op = self.bath.coupling_operator
-            _new_sys_correlations = \
-                self._process_tensor.calc_correlations(coup_op,correlation_set)
+        dt = self._process_tensor.dt
+        last_time = corr_mat_dim*dt
+        tlist = np.arange(0,last_time+dt,dt)
```
Comment (gefux) [inline oqupy/bath_dynamics.py:137]: Minor thing on style: I think one is supposed to put whitespaces after commas.
`tlist = np.arange(0,last_time+dt,dt)` -> `tlist = np.arange(0, last_time+dt, dt)`

This concerns a lot of lines in the code, would be great if you have a rough look through. Note: Editors sometimes have an auto-formatting function that might help. The vscode editor has.
I shall configure pylint accordingly (I haven't found out why it doesn't complain about this at the moment).

**Review by gefux — CHANGES_REQUESTED** [review]
Hi Dom! Thank you for this. It is great to see this working.
I've made a few minor comments.

File: oqupy/bath_dynamics.py [hunk 2]
```diff
@@ -91,47 +119,70 @@ def bath_occupation(self,
...
+        if freq == 0:
+            return tlist,np.zeros(len(tlist))
```
[the remark this answers is not in the document]
Comment (djgribben) [reply oqupy/bath_dynamics.py]: I agree with this, in a sense the change in occupation of the zero frequency mode is undefined so it makes sense to assign it nan

[the remark this answers is not in the document]
Comment (djgribben) [reply oqupy/bath_dynamics.py]: I've implemented this change

File: oqupy/bath_dynamics.py (hunk 1 above)
Comment (djgribben) [reply oqupy/bath_dynamics.py]: The auto-formatting in Spyder made some changes I didn't like so I've gone through by hand and made quite a few changes, hopefully caught them all!

Comment (gefux) [reply oqupy/bath_dynamics.py]: That's great. Thank you!

**Review by gefux — APPROVED** [review]

**Pull request merged**